### PR TITLE
Fix deps - fix tests

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Comment by xabbuh** [comment]
Not all the updated Notifier transport had failing tests before (probably because they only provide chatter integrations). Do we want to update their `composer.json` files anyway?

File: src/Symfony/Bundle/FrameworkBundle/composer.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "symfony/mailer": "^5.4|^6.0",
         "symfony/messenger": "^6.1",
         "symfony/mime": "^5.4|^6.0",
-        "symfony/notifier": "^5.4|^6.0",
+        "symfony/notifier": "^6.2",
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/composer.json]: Why is this needed?

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: It's not you're right.

**Comment by nicolas-grekas** [comment]
> Not all the updated Notifier transport had failing tests before (probably because they only provide chatter integrations). Do we want to update their composer.json files anyway?

Yes because it's too time consuming to bump selectively and it wouldn't be worth it :)
